e2e-tests/insurance/cypress/e2e/journeys/your-business/nature-of-business/nature-of-business-page.spec.js

```diff
@@ -97,31 +96,11 @@ context('Insurance - Your business - Nature of your business page - As an Export
       cy.checkText(field.suffix(), FIELDS.NATURE_OF_YOUR_BUSINESS[fieldId].SUFFIX);
     });
 
-    it('should display save and go back button', () => {
-      cy.checkText(saveAndBackButton(), BUTTONS.SAVE_AND_BACK);
-    });
```

Comment:
I imagine the save and back button/functionality should remain?

Reply:
its part of the next test under this.  Was duplicated